release/__init__.py

```diff
@@ -4,3 +4,4 @@
 #
 # The full license is in the file LICENSE.txt, distributed with this software.
 # -----------------------------------------------------------------------------
+from __future__ import annotations
```

Comment:
Why is this necessary or useful? The computational cost of computing annotations at runtime is irrelevant for the release scripts. Please revert.

Reply:
I added this for the sake of consistency across the codebase, so that we have the same semantics everywhere. Previously I was adding this on case-by-case basis, but tripping over differing semantics gets annoying pretty quickly. I didn't consider performance implications.

Reply:
If this is just about forward references we could continue to use strings and make a clean switch without `__future__` when 3.10 is the min version, instead of adding so much diff noise now.